### ISSUE TITLE
How To: Export Trimmed Clips as Individual Files

### ISSUE THREAD

**Issue by dryutsun** [jliljebl/flowblade-forum#47]
Hi All,

I am currently adjusting to the flowblade workflow. I am in the midst of analyzing a film shot by shot to get a deeper understanding of the editing decisions that went into its construction. I have taken the film and created proxies for them for performance improvements.

Therefore, I am trimming the clips in the timeline and renaming them in accordance to the scene. I was wondering if there was a way to export the clips that I have trimmed in my project into individual clips that I can reimport later. In other words, is there a way to export the individual clips to a folder?

**Comment by jliljebl** [comment]
No, the feature requested does not exist. The workflow you describe seems like something that is not common enough to warrant spending development time on. These are the kind of features that we can accept as code contributions but unfortunately will not work on as there are many other ways to improve the application that are considered more useful.
